HaskellNet.cabal

```diff
@@ -54,12 +60,13 @@ Library
     Network.HaskellNet.Debug
 
   Other-modules:
+    Network.Compat
     Text.Packrat.Pos
     Text.Packrat.Parse
 
   Build-Depends:
-    base >= 4.3 && < 4.13,
-    network >= 2 && < 3,
+    base >= 4.3 && < 4.14,
+    network >= 2.6.3.1 && < 3.2,
```

Comment:
https://hackage.haskell.org/package/network-2.6.3.1 is an most recent `network` which still supports GHC-7.0

It's released about three years ago, *at Sun Jul 31 03:48:32 UTC 2016*.

Reply:
Thanks for this info!